src/NvimClient.APIGenerator/NvimAPIGenerator.cs

```diff
@@ -140,7 +140,8 @@ private object GetExtensionType(MessagePackExtendedTypeObject msgPackExtObj)
   public class {eventName}EventArgs : EventArgs
   {{
 {
-      string.Join("", uiEvent.Parameters.Select(param => {
+      string.Join("", uiEvent.Parameters.Select(param =>
+      {
```

Comment:
sigh, are they really forcing this? would be nice to have opening braces at EOL.

Reply:
I can check if there's an option for it, but yes, afaik that is the idiomatic formatting used in C#.

Reply:
Okay so there *is* a config option for this (`csharp_new_line_before_open_brace = none` in `.editorconfig`). Want me to set that?